.obs/helper_scripts/listimages.sh

```diff
@@ -38,8 +38,14 @@ done
 echo "${REGISTRY}/rancher/elemental-operator:${OPERATOR}"
 echo "${REGISTRY}/rancher/seedimage-builder:${OPERATOR}"
 
-echo "${REGISTRY}/rancher/elemental-channel:${OPERATOR}"
-echo "${REGISTRY}/rancher/elemental-rt-channel:${OPERATOR}"
+# Note sort -V is a natural numbering sort, not semver
+higher_ver=$(printf '%s\n' "${OPERATOR}" "1.6.0" | sort -rV | head -n1)
+if [ "${higher_ver}" == "1.6.0" ]; then
```

Comment:
This will not really scale in the future specially when we start providing many different channels per distro stream. So this is just a little patch to keep this script consistent.